hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestUtils.java

```diff
@@ -233,6 +233,10 @@ public static HoodieTableMetaClient.TableBuilder getMetaClientBuilder(HoodieTabl
       builder.setKeyGeneratorType(properties.getProperty(HoodieTableConfig.KEY_GENERATOR_TYPE.key()));
     }
 
+    if (properties.containsKey("hoodie.write.table.version")) {
```

Comment:
lets try to use the variables already defined for this. also in L 237

Reply:
The config variables are not available in this module